### PR TITLE
Cranlogs ki

### PR DESCRIPTION
**Pull request by kweav**
Edits made in tracking CRAN downloads: 

- the dependency within the Rmd was switched from using the `dlstats` package to use `cranlogs`.
- The function used by `cranlogs` returns download data for every day (from a given start date)
-- so now the data is aggregated to sum downloads by month for each package
-- I also filter out months with 0 downloads for a given package since I use the earliest package's first publishing date to pull data for all packages
- the plot has been modified to 
-- use a color-blind friendly palette
-- update axis labels
-- include vertical dashed lines to highlight when packages were first published on CRAN
-- show month/year labels every 4 tick labels

### CODE REVIEW

File: itn-eval-metrics.Rmd [hunk 1]
```diff
@@ -535,10 +536,34 @@ ggplot2::ggsave(file.path("plots", "itn_software_website_traffic.png"), width =
...
+download_stats <- cranlogs::cran_downloads(packages = c("ottrpal", "conrad", "ari", "text2speech"), from = "2017-08-31", to = "last-day") #first version of ari published on 2017-08-31
 
- #download_stats %>% dplyr::group_by(package) %>%
- #  dplyr::summarize(download_total = sum(downloads))
+download_stats %>% dplyr::summarize(download_total = sum(count))
+
+download_stats %>% dplyr::group_by(package) %>%
+   dplyr::summarize(download_total = sum(count))
+
+download_stats_to_plot <- download_stats %>% separate(date, into=c("year", "month name", "day"), sep = "-") %>% unite("Month", c("year", "month name"), sep='-', remove=TRUE) %>%  #cran_downloads returns the daily downloads, and so doing some prep work to turn that into monthly downloads
```
Comment (cansavvy) [inline itn-eval-metrics.Rmd:546]: This reminds me I should run this whole thing through `styler::style_file()` But I can do that on another branch.

File: itn-eval-metrics.Rmd [hunk 2]
```diff
@@ -535,10 +536,34 @@ ggplot2::ggsave(file.path("plots", "itn_software_website_traffic.png"), width =
...
+  group_by(Month, package) %>% summarise(monthly_downloads = sum(count)) %>% #summarize monthly downloads by package
+  filter(monthly_downloads > 0) #drop the 0's 
+
+xlabel_view <- c(rep(c("black", "transparent", "transparent", "transparent"), 41), "black", "transparent") #166 rows
+cc <- rev(c("#fde725", "#addc30", "#5ec962", "#28ae80", "#21918c", "#2c728e", "#3b528b", "#472d7b", "#440154"))
+viridis_cc <- c("#440154", "#2c728e", "#28ae80", "#addc30")
```
Comment (cansavvy) [inline itn-eval-metrics.Rmd:552]: I might switch the other plots to this palette!

File: itn-eval-metrics.Rmd [hunk 3]
```diff
@@ -535,10 +536,34 @@ ggplot2::ggsave(file.path("plots", "itn_software_website_traffic.png"), width =
...
+
+ggplot(download_stats_to_plot, aes(Month, monthly_downloads, group=package, color = package)) + 
+  geom_line() + 
+  geom_point() +
+  scale_colour_manual(values=viridis_cc) +
+  theme(panel.background = element_blank(), panel.grid = element_blank()) +
+  geom_vline(aes(xintercept = "2019-05"), linetype='dashed', color = '#addc30') + #text2speech published date
```
Comment (cansavvy) [inline itn-eval-metrics.Rmd:559]: Love this!

Comment (cansavvy) [reply itn-eval-metrics.Rmd:559]: We should add these same lines for when courses were published. (again that can be on another branch)

File: itn-eval-metrics.Rmd [hunk 4]
```diff
@@ -535,10 +536,34 @@ ggplot2::ggsave(file.path("plots", "itn_software_website_traffic.png"), width =
...
+  geom_vline(aes(xintercept="2022-02"), linetype='dashed', color = '#28ae80') + #ottrpal published date 
+  geom_vline(aes(xintercept="2023-07"), linetype='dashed', color = '#2c728e') + #conrad published date
+  theme(axis.text.x = element_text(angle = 90)) +
+  theme(axis.text.x=element_text(color= xlabel_view)) + #clean up x-axis labels
+  ylab("Monthly Downloads")
+  
+ggplot2::ggsave(file.path("plots", "itn_software_cran_downloads_ki.png"), width=10, height=5)
```
Comment (cansavvy) [inline itn-eval-metrics.Rmd:566]: Beautiful!

**Review by cansavvy — APPROVED** [review]
Beautiful! Love it! Thanks for doing this! Feel free to file PRs that make other plots better if you feel so inspired (but I know you have other things to do too).

**Comment by cansavvy** [comment]
Feel free to merge whenever, @kweav You do the honors!

File: itn-eval-metrics.Rmd (hunk 3 above)
Comment (kweav) [reply itn-eval-metrics.Rmd:559]: I can work on that!

**Pull request merged**